config/experimental/s3configsource/config.go

```diff
@@ -0,0 +1,19 @@
+package s3configsource
+
+import (
+	splunkprovider "github.com/signalfx/splunk-otel-collector"
+)
+
+type Config struct {
+
+	//AWS variables that likely won't be used but nice to have in case of future modification
+	*splunkprovider.Settings
+
+	Region string `mapstructure:"region"`
+
+	Bucket string `mapstructure:"bucket"`
+
+	Key string `mapstructure:"key"`
+
+	VersionId string `mapstructure:"version_id"`
```

Comment:
Add comments to explain what each field is.
Also nit: a bunch of unnecessary blank lines.

Reply:
VersionID, read more at https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

Reply:
Changed it, but I think it is more confusing to have it as VersionID as the aws sdk asks for the data to be passed in in an object where the field is called VersionId